### DISCUSSION TITLE
Private forks created by Classroom don't have actions enabled

### DISCUSSION THREAD

**Discussion by mainland** [community/community#102536] [GitHub Education]
### Hi everyone,

I have switched to using the new forking method with Classroom. The sync functionality is great for making sure students can easily get assignment updates, but there is a big downside: private repository forks have actions disabled by default. I use GitHub actions to run a test suite every time a student pushes to their repository, but with the new fork model, this doesn't happen.

Even worse, there is no way to enable actions on a forked repository using the REST API. This means I have to manually click on hundreds of repositories to enable actions.

Yes, I know it *looks* like the REST API provides this functionality, but it does not. Maybe the REST API hasn't yet caught up to the new multiple-forks-in-org feature.

Could Classroom automatically enable actions on private forks?

**Comment by gpincheiraa** [comment]
Same problem here.
In my case I have the assignments with a lot of tests

<img width="400" alt="Screen Shot 2024-01-30 at 09 14 34" src="https://github.com/community/community/assets/5608336/eb219bf7-1226-4406-96a8-30b9b5936219">

When the students accepts the assignment, the tests are not added with Github Classroom Bot in the Student's forked repository.
The following picture is how works BEFORE opt-in the new mode for assignments

<img width="1421" alt="Screen Shot 2024-01-30 at 09 18 56" src="https://github.com/community/community/assets/5608336/a70bc28b-95ad-4784-bbe5-dcaaf6570d3d">


Also the same @mainland problem in the Student's repository after forks, because Actions is not enabled by default.

**Comment by MisterBrash** [comment]
In my case, my student accounts don't see a "Sync Fork" button when I update the base code or README file. They have to go through the multiple clicks of a pull request and merge. This wouldn't be so problematic if it weren't a high school class who are completely new to GitHub (and programming).

**Comment by samus-aran** [comment]
Hi @mainland 👋 

We have an active discussion regarding this topic (enabling workflows on student forked repos). You can see the [latest updates and share your experiences here.](https://github.com/orgs/community/discussions/106545) 👈

**Reply by samus-aran** [reply]
Hi everyone 👋 

We have an update on **assignment forks will now have workflows enabled by default**. Please [check out the update comment here for additional information](https://github.com/orgs/community/discussions/106545#discussioncomment-8647837).
